test/test_sox_compatibility.py

```diff
@@ -9,9 +9,27 @@
 
 
 @common_utils.skipIfNoSoxBackend
-class TestFunctionalFiltering(common_utils.TorchaudioTestCase):
+class TestFunctionalFiltering(common_utils.TempDirMixin, common_utils.TorchaudioTestCase):
     backend = 'sox'
 
+    def setUp(self):
+        common_utils.TempDirMixin.setUp(self)
+        common_utils.TorchaudioTestCase.setUp(self)
+
+        NOISE_SAMPLE_RATE = 44100
```

Comment:
Can you use 8000 Hz? though 44100 Hz is common in audio, many speech-related tasks only use 16k or 8k Hz and 8k Hz will reduce test run time.

Reply:
Sox `deemph` seems to only work on 44.1kHz  or 48kHz

`deemph: sample rate must be 44100 (audio-CD) or 48000 (DAT)`